packages/vite/src/node/plugins/css.ts

```diff
@@ -684,9 +684,10 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
             `document.head.appendChild(${style});`
           const wrapIdx = code.indexOf('System.register')
           const executeFnStart =
-            code.indexOf('{', code.indexOf('execute:', wrapIdx)) + 1
+            wrapIdx >= 0 ? code.indexOf('execute:', wrapIdx) : 0
+          const injectionPoint = code.indexOf('{', executeFnStart) + 1
           const s = new MagicString(code)
-          s.appendRight(executeFnStart, injectCode)
+          s.appendRight(injectionPoint, injectCode)
```

Comment:
It seems this breaks the IIFE format. The output function is not in strict mode.
```js
(function() {
  var __vite_style__ = document.createElement("style");
  __vite_style__.textContent = "/* omit */";
  document.head.appendChild(__vite_style__);
  "use strict";
```
I think we need to inject the styles after the `"use strict"` if `System.register` wasn't found. (It was broken by #13266)

Reply:
Also the UMD output is a bit weird.
```js
(function(factory) {
  var __vite_style__ = document.createElement("style");
  __vite_style__.textContent = "/* omit */";
  document.head.appendChild(__vite_style__);
  typeof define === "function" && define.amd ? define(factory) : factory();
})(function() {
  "use strict";
```
I understand this would work. But I guess it's better to inject the styles after the `"use strict"` so that bundles can recognize the output file is written in UMD format.